debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java

```diff
@@ -157,6 +159,31 @@ public void insertsWithoutPks() throws Exception {
         }
     }
 
+    @Test
+    public void insertsNumericPk() throws Exception {
+        Testing.Print.enable();
+
+        try (final JdbcConnection connection = databaseConnection()) {
+            populateTable(connection, "s1.anumeric");
+        }
+        startConnector();
+
+        sendAdHocSnapshotSignal("s1.anumeric");
+
+        Thread.sleep(5000);
```

Comment:
Can we avoid this, in favour of something based on Awaitility?

Reply:
Superfluous, line removed.